WebAPI/Startup.cs

```diff
@@ -310,6 +310,8 @@ private MongoDbIdentityConfiguration ConfigureIdentity()
                     ConnectionString = _runner.ConnectionString + _swapifyDbSettings.DatabaseName,
                     DatabaseName = _swapifyDbSettings.DatabaseName
                 };
+                _logger.LogInformation($"Database connection string: {configuration.MongoDbSettings.ConnectionString} ");
```

Comment:
I'm note sure, but if there is a password inside the connection string, it should not be logged...

Reply:
I forget, we are not using password for database, but anyway, it can be changed and then we will log it here...